src/components/Avatar/Avatar.vue

```diff
@@ -455,10 +454,6 @@ export default {
 				links[0].focus()
 			}
 		},
-		handlePopoverAfterHide() {
-			// bring focus back to the trigger
```

Comment:
This is still a case that should be preserved when closing the menu with the keyboard I guess. This was introduced with https://github.com/nextcloud/nextcloud-vue/pull/1854 for accessibility reasons.